Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -47,6 +49,9 @@ Neos:
 
         'postprocess':
           chain:
+            'prepareSession':
```

Comment:
Are we still "preparing" the session at this stage? Maybe `'sendSession'` or similar might work better here.